internal/storage/db.go

```diff
@@ -76,3 +81,43 @@ func (s *DBStorage) AssignTokens(ctx context.Context, issuanceWeek, firstAutomat
 	_, err = s.DBS.DBS().Writer.ExecContext(ctx, assignTokensQuery, issuanceWeek, weekLimit.String())
 	return err
 }
+
+var tokensPerWeekQuery = `
+SELECT 
+	SUM(
+		r.streak_tokens +
+		r.synthetic_device_tokens +
+		r.aftermarket_device_tokens
+	)::numeric / NULLIF(SUM(
+        r.streak_points + 
+        r.integration_points + 
```

Comment:
Uhhh should this be here alongside aftermarket_ and synthetic_ ? That strikes me as odd